developers_affiliations8.txt

```diff
@@ -11390,6 +11390,8 @@ xrk123: 3104867938!qq.com, xrk123!users.noreply.github.com
 	Inspur Group
 xrl: xrl!users.noreply.github.com, xrlange!gmail.com
 	Fanpics
+xrmx: riccardo.magliocchetti!elastic.co, riccardo.magliocchetti!gmail.com
```

Comment:
```suggestion
xrmx: riccardo.magliocchetti!elastic.co, riccardo.magliocchetti!gmail.com, xrmx!users.noreply.github.com
```

I think the `xrmx!users.noreply.github.com` is being used when you interact through the GitHub API, so let's include it here as well.